apps/state_mediator/test/state_mediator/integration/gtfs_test.exs

```diff
@@ -61,19 +61,19 @@ defmodule StateMediator.Integration.GtfsTest do
       assert_first_last_stop_id("CR-Haverhill", "place-north", "place-WR-0329")
       assert_first_last_stop_id("CR-Lowell", "place-north", "place-NHRML-0254")
       assert_first_last_stop_id("CR-Kingston", "place-sstat", "place-KB-0351")
-      assert_first_last_stop_id("Green-B", "place-pktrm", "place-lake")
-      assert_first_last_stop_id("Green-C", "place-north", "place-clmnl")
-      assert_first_last_stop_id("Green-D", "place-gover", "place-river")
-      assert_first_last_stop_id("Green-E", ["place-north", "place-lech"], "place-hsmnl")
+      assert_first_last_stop_id("Green-B", ["place-pktrm", "place-gover"], "place-lake")
+      assert_first_last_stop_id("Green-C", ["place-gover", "place-north"], "place-clmnl")
+      assert_first_last_stop_id("Green-D", ["place-gover", "place-north"], "place-river")
+      assert_first_last_stop_id("Green-E", ["place-north", "place-unsqu"], "place-hsmnl")
```

Comment:
Why are these in addition to the existing stop IDs? I don't quite understand what it means for their to be two first stops on a route, despite `assert_first_last_stop_id` supporting this functionality.